apisix/plugins/example-plugin.lua

```diff
@@ -109,4 +110,28 @@ function _M.access(conf, ctx)
 end
 
 
+local function hello()
+    local args = ngx.req.get_uri_args()
+    if args["json"] then
+        return 200, {msg = "world"}
+    else
+        return 200, "world\n"
+    end
+end
+
+
+function _M.control_api(ver)
```

Comment:
I did not understand why we need this feature. 

I think you need to write a proposal first.

Reply:
I already wrote it https://github.com/apache/apisix/issues/2798 a month ago.
And https://github.com/apache/apisix/issues/3022 shows why we need this feature.

Reply:
thx, got it